src/components/Banners/CollateralClaim.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Text, Card, Button } from '@makerdao/ui-components-core';
+import useLanguage from 'hooks/useLanguage';
+import useMaker from 'hooks/useMaker';
+
+const CollateralClaim = ({ colName, amount, symbol }) => {
+  const { lang } = useLanguage();
+  const { maker, newTxListener } = useMaker();
+
+  const frobToOwnUrn = async () => {
+    //TODO need to fix re-render infinite loop to pass in real props
+    const cdpId = 183;
+    const amount = 0.08;
+
```

Comment:
note these are hardcoded at the moment, so you can view [CDP 183](http://localhost:3000/borrow/183?network=kovan) to repro the issue (or hardcode your own values)